tasks/partial_execution.yml

```diff
@@ -0,0 +1,110 @@
+---
+- name: Initial validations and cleanups
+  block:
+    - name: Install required packages for oVirt Hosted Engine deployment
+      import_tasks: install_packages.yml
+
+    - name: System configuration validations
+      include_tasks: "{{ item }}"
+      with_fileglob: "pre_checks/*.yml"
+
+    - name: Clean environment before deployment
+      import_tasks: initial_clean.yml
+  tags: [initial_clean, bootstrap_local_vm, never]
+
```

Comment:
redundant line

Reply:
I explicitly added double empty lines to keep differently tagged sections more visible